### PR TITLE
Favicons

### PR DESCRIPTION
**Pull request by BryanGK**
This PR adds the correct favicons to the application.

- new icons in static folder

### CODE REVIEW

File: app/pages/_app.tsx [hunk 1]
```diff
@@ -64,6 +65,34 @@ const formStyle = {
 export default function App({ Component, pageProps }) {
   return (
     <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (wenzowski) [inline app/pages/_app.tsx:69]: Is Next throwing any warnings about putting `<meta>` here?

File: app/pages/_app.tsx [hunk 2]
```diff
@@ -64,6 +65,34 @@ const formStyle = {
...
+        <meta charSet="utf-8" />
```
Comment (wenzowski) [inline app/pages/_app.tsx:70]:
```suggestion
        <meta charset="utf-8" />
```

**Review by wenzowski — CHANGES_REQUESTED** [review]

File: app/pages/_app.tsx (hunk 1 above)
Comment (BryanGK) [reply app/pages/_app.tsx:69]: Next was throwing errors having metadata in _document.tsx and recomended it to be put in _app.tsx

File: app/pages/_app.tsx (hunk 2 above)
Comment (BryanGK) [reply app/pages/_app.tsx:70]: Changing to this throws an error.

File: app/pages/_app.tsx (hunk 1 above)
Comment (wenzowski) [reply app/pages/_app.tsx:69]: perfect

**Review by wenzowski — APPROVED** [review]

**Pull request merged**